css/buttons.css

```diff
@@ -0,0 +1,75 @@
+.btn {
+    outline: none;
+    border: none;
```

Comment:
Suggestion: You can remove this as you are defining border again. The border value is none by default. Let me know about your thoughts on this.

Reply:
yeah will refactor all at once. Thanks !!